controllers/userController.js

```diff
@@ -2,48 +2,56 @@ const ApiError = require("../error/ApiError");
 const bcrypt = require("bcrypt");
 const jwt = require("jsonwebtoken");
 const { User, Basket } = require("../models/models");
+const { USER } = require("../utils/errors");
 
-const generateJwt = (id, email, role) => {
-  return jwt.sign({ id, email, role }, process.env.SECRET_KEY, {
+const generateJwt = (id, email, role, name) => {
+  return jwt.sign({ id, email, role, name }, process.env.SECRET_KEY, {
     expiresIn: "24h",
```

Comment:
not so critical, but also might be added from .env and have a default value. And 24h? Why so short life? Let it live a month at least :)